<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_arrow">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>36</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_floor">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>242</y>
     <width>200</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>428</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_destination">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>448</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
